From 50b617e69aa84d9d7ea484a1956caf610e276e3b Mon Sep 17 00:00:00 2001
From: Johannes Thoma <johannes@johannesthoma.com>
Date: Tue, 16 Oct 2018 18:41:50 +0200
Subject: [PATCH 4/4] Do not check assertion on dec peerdevice

Something with our 'drbd_polymorph_printk' does not work quite right.
---
 drbd/drbd_receiver.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drbd/drbd_receiver.c b/drbd/drbd_receiver.c
index f5ac522..10931d1 100644
--- a/drbd/drbd_receiver.c
+++ b/drbd/drbd_receiver.c
@@ -3076,7 +3076,7 @@ void drbd_cleanup_peer_requests_wfa(struct drbd_device *device, struct list_head
 	list_for_each_entry_safe(struct drbd_peer_request, peer_req, pr_tmp, cleanup, wait_for_actlog) {
 		atomic_sub(interval_to_al_extents(&peer_req->i), &device->wait_for_actlog_ecnt);
 		atomic_dec(&device->wait_for_actlog);
-		dec_unacked(peer_req->peer_device);
+		__dec_unacked(peer_req->peer_device);
 		list_del_init(&peer_req->wait_for_actlog);
 		drbd_may_finish_epoch(peer_req->peer_device->connection, peer_req->epoch, EV_PUT | EV_CLEANUP);
 		drbd_free_peer_req(peer_req);
-- 
2.17.0

